src/app/tabs/tabbed-menu.js

```diff
@@ -1,52 +1,97 @@
 var yo = require('yo-yo')
-var helper = require('../../lib/helper')
+var csjs = require('csjs-inject')
+var remixLib = require('remix-lib')
 
-var css = require('./styles/tabbed-menu-styles')
+var styles = require('../ui/styles-guide/theme-chooser').chooser()
 
-class TabbedMenu {
-  constructor (api = {}, events = {}, opts = {}) {
-    var self = this
-    var tabView = document.createElement('ul')
-    this.tabView = tabView
-    this.events = events
-    this.tabs = {}
-    this.contents = {}
-
-    events.app.register('debuggingRequested', () => {
-      self.selectTab(tabView.querySelector('li.debugView'))
-    })
+var EventManager = remixLib.EventManager
 
-    events.rhp.register('switchTab', tabName => {
-      self.selectTab(tabView.querySelector(`li.${tabName}`))
-    })
+module.exports = class TabbedMenu {
+  constructor (api = {}, events = {}, opts = {}) {
+    const self = this
+    self.event = new EventManager()
+    self._api = api
+    self._events = events
+    self._view = { el: null, viewport: null, tabs: {}, contents: {} }
+    self.data = {}
```

Comment:
seems you don't use `data` neither `_components` . perhaps not necessary to declare them